src/clojure_lsp/db.clj

```diff
@@ -16,7 +16,7 @@
 (defn ^:private get-sqlite-db-file-path [project-root]
   (let [configured (some-> (get-in @db [:settings :sqlite-db-path])
                            io/file)
-        default (io/file (str project-root) ".lsp" "sqlite.db")
```

Comment:
Not sure this should be removed, otherwise how can we make sure the db will be saved on project root?

Reply:
project root will be added in following .isAbsolute test.
Otherwise, two project root will added before .lsp in windows, because /user/project/.lsp/sqlite.db is considered as a relative path, thus getAbsolutePath will turn it to: D:/user/project/user/project/.lsp/sqlite.db

Reply:
like this:
```
lein test :only clojure-lsp.db-test/sqlite-db-file-setting

FAIL in (sqlite-db-file-setting) (db_test.clj:18)
when not set
expected: "D:\\user\\project\\.lsp\\sqlite.db"
  actual: "D:\\user\\project\\user\\project\\.lsp\\sqlite.db"
    diff: - "D:\\user\\project\\.lsp\\sqlite.db"
          + "D:\\user\\project\\user\\project\\.lsp\\sqlite.db"
```

Reply:
I tested on linux and it works, I'm just curious how io/file return the path to the project + .lsp/config.edn, how it guesses the project root correctly?

Reply:
Yes. That's the problem. The driver in absolute path may change according to project location, under C: or under D:

Reply:
I see, so this could not be the same for other computes?
I think we should find some other way so, otherwise we can save the db on another path or get a wrong path

Reply:
Hey, but we provide `project-path` as the parameter, Windows user should specify correct project root with driver characters.

Reply:
yeah, use the project-root would be perfect, the current code it's using it

Reply:
the project-root comes from the client, from emacs, lsp-mode sends the project-root correctly already

Reply:
So, @kimim can we somehow keep using the project-root?